webapp/channels/src/workers/sw.js

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// Update this every time code is changed.
```

Comment:
Does this refer to just the code for the service worker, or does it refer to any code in the app?

Reply:
Just the code for the service worker.